packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx

```diff
@@ -126,7 +122,12 @@ describe('CFDAccountCard', () => {
                 descriptor={synthetic_descriptor}
                 title='Derived'
                 existing_accounts_data={[mt5_acc]}
-            />
+            />,
+            {
+                wrapper: ({ children }: { children: JSX.Element }) => (
+                    <CFDProviders store={mockStore(mock_connect_props)}>{children}</CFDProviders>
+                ),
+            }
```

Comment:
Is it possible to reuse this code snippet? it is being duplicated all over this file!

Reply:
Done